### PR TITLE
Adding text summarisation

### LINKED ISSUE #5: Add support for text summarisation
#### Is your feature request related to a problem? Please describe.
We currently have support for image and text classification it would be good to have support for summarisation too

### PR DESCRIPTION
**Pull request by gagan3012**
#### Changes

Fixes #5  (issue)

Adding Text summarisation to Gradsflow

#### Type of change

- [x] Documentation Update
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: tests/autoclassifier/test_sum.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+def test_model():
+    datamodule = MagicMock()
+    model = AutoSummarisation(
+        datamodule,
+        max_epochs=1,
+        timeout=5,
+        suggested_backbones="sshleifer/distilbart-cnn-12-6",
+        n_trials=1,
+    )
+    model.hp_tune()
```
Comment (aniketmaurya) [inline tests/autoclassifier/test_sum.py:16]: `model.hp_tune()` will try to train the model with MagicMock class. We can remove it from here.

**Review by aniketmaurya — APPROVED** [review]
Looks good 😀

**Comment by aniketmaurya** [comment]
Great work @gagan3012 ! Could you rename `AutoSummarisation` to `AutoSummarization`? Gradsflow should follow the same naming as Flash and Huggingface.

File: tests/autoclassifier/test_sum.py (hunk 1 above)
Comment (gagan3012) [reply tests/autoclassifier/test_sum.py]: Should we define datamodule as none then?

Comment (gagan3012) [reply tests/autoclassifier/test_sum.py]: Oh nevermind I got what you mean now!

**Comment by gagan3012** [comment]
@aniketmaurya Hey I think its ready

**Pull request merged**